dev/scripts/release/analyze.sh

```
@@ -0,0 +1,284 @@
+require_app_interface
+
+get_prod_ref() {
+  python3 - <<PYEOF
+with open("${DEPLOY_FILE}") as f:
+    content = f.read()
+in_prod = False
+for line in content.splitlines():
+    if 'hccm-prod.yml' in line:
+        in_prod = True
+        continue
+    if in_prod and line.strip().startswith('ref:') and '$ref' not in line:
```

Shellcheck (0.11.0)

[warning] ref is referenced but not assigned.

(SC2154)